cli-feed-v3-3.json

```diff
@@ -0,0 +1,317 @@
+{
+  "tags": {
+    "v1": {
+      "release": "1.13.0",
+      "displayName": ".NET Framework",
+      "displayVersion": "v1",
+      "releaseQuality": "GA",
+      "targetFramework": ".NET Framework",
+      "capabilities": "",
+      "hidden": false
+    },
+    "v1-prerelease": {
+      "release": "1.13.0",
+      "displayName": ".NET Framework",
+      "displayVersion": "v1",
+      "releaseQuality": "Prerelease",
+      "targetFramework": ".NET Framework",
+      "capabilities": "",
+      "hidden": true
+    },
+    "v2": {
+      "release": "2.60.0",
+      "displayName": ".NET Core 2.2",
+      "displayVersion": "v2",
+      "releaseQuality": "GA",
+      "targetFramework": ".NET Core",
+      "capabilities": "",
+      "hidden": false
+    },
+    "v2-prerelease": {
+      "release": "2.60.0",
+      "displayName": ".NET Core 2.2",
+      "displayVersion": "v2",
+      "releaseQuality": "Prerelease",
+      "targetFramework": ".NET Core",
+      "capabilities": "",
+      "hidden": true
+    },
+    "v2-preview": {
+      "release": "2.60.0",
+      "displayName": ".NET Core 2.2",
+      "displayVersion": "v2",
+      "releaseQuality": "Preview",
+      "targetFramework": ".NET Core",
+      "capabilities": "",
+      "hidden": true
+    },
+    "v3-preview": {
+      "release": "3.18.0",
+      "displayName": ".NET Core 3.1",
+      "displayVersion": "v3",
+      "releaseQuality": "Preview",
+      "targetFramework": ".NET Core",
+      "capabilities": "",
+      "hidden": true
+    },
+    "v3-prerelease": {
+      "release": "3.19.0",
+      "displayName": ".NET Core 3.1",
+      "displayVersion": "v3",
+      "releaseQuality": "Prerelease",
+      "targetFramework": ".NET Core",
+      "capabilities": "",
+      "hidden": true
+    },
+    "v3": {
+      "release": "3.18.0",
+      "displayName": ".NET Core 3.1",
+      "displayVersion": "v3",
+      "releaseQuality": "GA",
+      "targetFramework": ".NET Core",
+      "capabilities": "",
+      "hidden": false
+    },
+    "v3-isolated": {
```

Comment:
We need to confirm that adding the `-isolated` is acceptable by everyone. I would imagine that because we are going with a new file completely, this would be ok. If right now tools expect a certain format, they can be updated as we only expect net5 to work from future release of the tools. Past releases would continue to use the old feed.